x/evm/types/config.go

```diff
@@ -2,18 +2,16 @@ package types
 
 // EVMConfig contains all EVM module configuration values
 type EVMConfig struct {
-	Name               string `mapstructure:"name"`
-	RPCAddr            string `mapstructure:"rpc_addr"`
-	WithBridge         bool   `mapstructure:"start-with-bridge"`
-	EnableRPCDetection bool   `mapstructure:"enable-rpc-detection"`
+	Name       string `mapstructure:"name"`
+	RPCAddr    string `mapstructure:"rpc_addr"`
+	WithBridge bool   `mapstructure:"start-with-bridge"`
 }
 
 // DefaultConfig returns a configuration populated with default values
 func DefaultConfig() []EVMConfig {
 	return []EVMConfig{{
-		Name:               "Ethereum",
-		RPCAddr:            "http://127.0.0.1:7545",
-		WithBridge:         true,
-		EnableRPCDetection: true,
+		Name:       "Ethereum",
+		RPCAddr:    "http://127.0.0.1:7545",
```

Comment:
Why do we have a default config at all? Would be good to remove it.

Reply:
It serves as an example. Default typically doesn't work but it is good doc.